### PR TITLE
docs: polish secret key implementation description

### PR DESCRIPTION
**Pull request by nisiyong**
## What's the purpose of this PR

polish secret key implementation description

## Which issue(s) this PR fixes:
Fixes https://github.com/apolloconfig/apollo/issues/4856

### CODE REVIEW

**Comment by nobodyiam** [comment]
Would you please also help to update the [en](https://github.com/apolloconfig/apollo/blob/master/docs/en/usage/other-language-client-user-guide.md) doc?

**Comment by nisiyong** [comment]
> Would you please also help to update the [en](https://github.com/apolloconfig/apollo/blob/master/docs/en/usage/other-language-client-user-guide.md) doc?

Sure, Please take a look.

File: docs/en/usage/other-language-client-user-guide.md [hunk 1]
```diff
@@ -176,10 +176,10 @@ Apollo has added an access key mechanism since version 1.6.0, so that only authe
...
+| Header        | Value                                                        | Remarks                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| ------------- | ------------------------------------------------------------ |----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (nobodyiam) [inline docs/en/usage/other-language-client-user-guide.md:180]:
```suggestion
| ------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
```

File: docs/en/usage/other-language-client-user-guide.md [hunk 2]
```diff
@@ -176,10 +176,10 @@ Apollo has added an access key mechanism since version 1.6.0, so that only authe
...
+| Authorization | Apollo ${appId}:${signature}                                 | appId: The appId of the application, signature: Generate a signature by combining the access key with the current timestamp in milliseconds, as well as the path and query components of the visited URL. The specific implementation can be found in [Signature.signature](https://github.com/apolloconfig/apollo/blob/aa184a2e11d6e7e3f519d860d69f3cf30ccfcf9c/apollo-core/src/main/java/com/ctrip/framework/apollo/core/signature/Signature.java#L22) |
+| Timestamp     | Number of milliseconds elapsed from `1970-1-1 00:00:00 UTC+0` to now | See [System.currentTimeMillis](https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis())                                                                                                                                                                                                                                                                                                                                      |
```
Comment (nobodyiam) [inline docs/en/usage/other-language-client-user-guide.md:182]:
```suggestion
| Timestamp     | Number of milliseconds elapsed from `1970-1-1 00:00:00 UTC+0` to now | See [System.currentTimeMillis](https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#currentTimeMillis()) |
```

File: docs/en/usage/other-language-client-user-guide.md [hunk 3]
```diff
@@ -176,9 +176,9 @@ Apollo has added an access key mechanism since version 1.6.0, so that only authe
 
 Header information to be set.
 
-| Header        | Value                                                        | Remarks                                                      |
+| Header        | Value                                                        | Remarks                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (nobodyiam) [inline docs/en/usage/other-language-client-user-guide.md:179]:
```suggestion
| Header        | Value                                                        | Remarks                                                      |
```

**Review by nobodyiam — APPROVED** [review]
LGTM

**Pull request merged**